wrapper/mm.h

```diff
@@ -0,0 +1,70 @@
+/* SPDX-License-Identifier: (GPL-2.0 or LGPL-2.1)
+ *
+ * wrapper/mm.h
+ *
+ * Copyright (C) 2018 Francis Deslauriers <francis.deslauriers@efficios.com>
+ */
+
+#ifndef _LTTNG_WRAPPER_MM_H
+#define _LTTNG_WRAPPER_MM_H
+
+#include <linux/mm.h>
+#include <linux/oom.h>
+
```

Comment:
are we sure those 2 includes exist on older kernels ?

Reply:
I just checked.
oom.h was added in v2.6.19
mm.h was added before v2.6.12